app/ui-react/syndesis/src/modules/integrations/components/editor/choice/SelectChoiceConfigModePage.tsx

```diff
@@ -58,23 +59,27 @@ export interface ISelectChoiceConfigModePageProps extends IPageWithEditorBreadcr
  * **Warning:** this component will throw an exception if the route state is
  * undefined.
  */
-export class SelectChoiceConfigModePage extends React.Component<ISelectChoiceConfigModePageProps> {
+export class SelectChoiceConfigModePage extends React.Component<
+  ISelectChoiceConfigModePageProps
+> {
   public render() {
     return (
       <WithRouteData<ISelectConfigModeRouteParams, ISelectConfigModeRouteState>>
         {(params, state) => {
           const positionAsNumber = parseInt(params.position, 10);
           const options = [
             {
-              description: 'Provides preselected properties and condition operators for simple expressions.',
+              description:
+                'Provides preselected properties and condition operators for simple expressions.',
```

Comment:
Should we not be using i18n here? Or are we kind of putting that on the back burner for now?

Reply:
well sheesh I didn't even spot that, was just updating the usage of the component.

But now I can't unsee it, lemme tidy up what I'm doing and I'll add that change to this PR.

Reply:
K, updated that file in a separate commit.